R/post-body.R

```diff
@@ -24,3 +34,33 @@ parseBody <- function(body){
   }
   ret
 }
+
+
+#' @importFrom utils URLdecode
+#' @noRd
+parseMultipart <- function(body, boundary){
+  function(val, req, res, errorHandler){
+    tryCatch({
+      if (!requireNamespace("webutils", quietly = TRUE)) {
+        stop("The webutils package is not available but is required in order
+          to use the functionality to POST binary files",
+             call. = FALSE)
+      }
+      # Is there data in the request?
+      if (is.null(body) || length(body) == 0 || body == "") {
+        return(list())
+      }
+      parsed_binary <- webutils::parse_multipart(body, boundary)
```

Comment:
Are there limits built in to any of these on how big of a file we'd attempt to parse? Anything to prevent a client from sending a 10GB file and running the server out of RAM?